index.html

```diff
@@ -630,16 +630,16 @@ <h2 class="large section-title">Notre plan de commandites</h2>
                             </div><!--pricing wrapper-->
                         </div><!--price col-->
                         <div class="col-md-4">
-                            <div class="pricing-wrapper wow animated fadeInRight" data-wow-delay=".9s">
+                            <div class="pricing-wrapper price-popular wow animated fadeInUp" data-wow-delay=".6s">
                                 <div class="pricing-wrapper-inner">
-                                    <h1>i5</h1>
-                                    <h2>1000 $</h2>
+                                    <h1>i7</h1>
```

Comment:
Agreed! Also, IMHO, the X implies that a feature is not included. You know, something like this. How about we just remove the X (or replace it with something that doesn't imply a disadvantage)?
![2015-12-28-200724_664x239_scrot](https://cloud.githubusercontent.com/assets/8457808/12028036/a23e5ad2-ad9e-11e5-8ef2-bcfa76519d63.png)